app/views/admin/relays/_relay.html.haml

```diff
@@ -4,7 +4,7 @@
   %td
     - if relay.accepted?
       %span.positive-hint
-        = fa_icon('check')
+        = material_symbol 'check'
```

Comment:
Not really an issue, but the styling is inconsistent with both the old version and the new calls.